src/components/Content.tsx

```diff
@@ -15,6 +16,12 @@ export interface ContentProps<T extends ComponentType | undefined> {
 export const Content = (<T extends ComponentType | undefined>(
   {html, component, block = false, ...props}: ContentProps<T>
 ) => {
+  const typesetMath = useTypesetMath();
+
+  React.useEffect(() => {
+    typesetMath();
+  }, [html])
```

Comment:
this seems like it should be complaining about the missing dependency in the array